### DISCUSSION TITLE
FTP username password

### DISCUSSION THREAD

**Discussion by amboxer21** [aldostools/webMAN-MOD#469] [General]
Why is it so ****ing hard to figure out the ftp username and password? Or how to change it even.

**Comment by aldostools** [comment]
User name can be any name... the text is ignored.

The password stored in the config file /dev_hdd0/tmp/wm_config.bin

Delete that file and you will be able to access the ftp without restrictions. Or change it using the /setup.ps3
![image](https://user-images.githubusercontent.com/974897/105611959-0f19fd00-5d8f-11eb-9c1f-dc346409d7cd.png)

PS. Next time be kind. There is no need for swearing to ask for support..
